app/controllers/users_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
   private
 
     def user_params
-      params.require(:user).permit(:name, :email, :password,
+      params.require(:user).permit(:name, :email, :location, :password,
```

Comment:
I think Ankur has them split up on the web page so probably that. It's on the to-do list.